l10n_fi_invoice/i18n/sv.po

```diff
@@ -0,0 +1,278 @@
+msgid "<span class=\"offset-2\">Description</span>"
+msgstr "<span class=“offset-2”>Beskrivning</span>"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "<span>Date due</span>"
+msgstr "<span>Förf.dag</span>"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "<span>Invoice number</span>"
+msgstr "<span>Fakturanummer</span>"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "<span>Invoice reference</span>"
+msgstr "<span>Faktura referens</span>"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "<strong>Total</strong>"
+msgstr "<strong>Totalt</strong>"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Amount"
+msgstr "Belopp"
+
+#. module: l10n_fi_invoice
+#: model:ir.model.fields,field_description:l10n_fi_invoice.field_account_move__barcode_string
+msgid "Barcode String"
+msgstr "Streckkod"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Base"
+msgstr "Bas"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Business ID:"
+msgstr "FO-nummer:"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Date due"
+msgstr "Förf.dag"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Date fulfilled"
+msgstr "Datum uppfyllt"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Debtor"
+msgstr "Betalare"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Discount (%)"
+msgstr "Rabatt (%)"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "Domicile:"
+msgstr "Hemvist:"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_layout_footer_finnish
+msgid "Email:"
+msgstr "E-post:"
+
+#. module: l10n_fi_invoice
+#: model:ir.actions.report,name:l10n_fi_invoice.report_invoice_finnish
+msgid "Finnish Invoice"
+msgstr "Finska faktura"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "From account No."
+msgstr "Från konto nr"
+
+#. module: l10n_fi_invoice
+#: model_terms:ir.ui.view,arch_db:l10n_fi_invoice.report_invoice_finnish_document
+msgid "IBAN"
+msgstr "IBAN"
+
+
+#. module: l10n_fi_invoice
+#: model:ir.model.fields,field_description:l10n_fi_invoice.field_account_journal__include_on_invoice
+msgid "Include on Invoice"
+msgstr ""
```

Comment:
Some Swedish translations missing. Could you help with this, @aisopuro?

Reply:
No need anymore. This field was removed from the module.

Reply:
Yes, this particular string, but I counted at least 4 missing translations further down in the file.